rtv/submission.py

```diff
@@ -208,4 +215,48 @@ def draw_submission(win, data):
         text = Symbol.clean('{score} {comments}'.format(**data))
         win.addnstr(row, 1, text, n_cols, curses.A_BOLD)
 
-        win.border()
\ No newline at end of file
+        win.border()
+
+    def add_comment(self):
+        """
+        Add a comment on the submission if a header is selected.
+        Reply to a comment if the comment is selected.
+        """
+
+        if not self.reddit.is_logged_in():
+            display_message(self.stdscr, ["You are not logged in!"])
+            return
+
+        cursor_position = self.nav.absolute_index
+        if (self.content.get(cursor_position)['type'] != 'Comment') \
+           & (self.content.get(cursor_position)['type'] != 'Submission'):
+            display_message(self.stdscr, ['Expand the comments first!'])
+            return
+
+        n_rows, n_cols = self.stdscr.getmaxyx()
+        box_height = n_rows/2
+
+        attr = curses.A_BOLD | Color.CYAN
+        prompt = 'Enter comment: ESC to cancel, Ctrl+g to submit'
+        prompt = '-'*((n_cols-len(prompt))/2) + prompt \
+                 + '-'*((n_cols-len(prompt)+1)/2)
```

Comment:
This looks really nice, good idea putting the text in the middle of the line! Have you though about using the curses ACS_HLINE instead of just a dash? I think it would look a little cleaner to have a solid line across as opposed to a dashed line.

Two ways you could do it:
1. Use [window.hline](https://docs.python.org/2/library/curses.html#curses.window.hline) and just write the text on top of the line. Downside is that ``hline`` only draws white lines.
2. Do something similar to what I did to draw the vertical bars
```python
        for y in range(n_rows):
            x = 0
            # http://bugs.python.org/issue21088
            if (sys.version_info.major,
                sys.version_info.minor,
                sys.version_info.micro) == (3, 4, 0):
                x, y = y, x
            win.addch(y, x, curses.ACS_VLINE, attr)
```